src/pages/ProfilePage/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import smilies from 'assets/smilies.svg';
+import defaultProfile from 'assets/default_profile.svg';
+
+import { getProfile, updatePassword, deleteAccount } from 'redux/actions/userActions';
+import FormInput from 'components/common/formInput/FormInput';
+import useForm from 'hooks/useForm';
+import Button from 'components/common/button/Button';
+import withMap from 'components/HOCs/withMap';
```

Comment:
Just to have the code more organized it is good to have all imports related to each other together. For example, as you did when importing the assets you can do it with the components.